azurerm/provider.go

```diff
@@ -327,8 +327,14 @@ func Provider() terraform.ResourceProvider {
 		"azurerm_private_dns_cname_record":                                               resourceArmPrivateDnsCNameRecord(),
 		"azurerm_public_ip":                                                              resourceArmPublicIp(),
 		"azurerm_public_ip_prefix":                                                       resourceArmPublicIpPrefix(),
+		"azurerm_recovery_network_mapping":                                               resourceArmRecoveryServicesNetworkMapping(),
```

Comment:
I think you need to rebase/merge?

Reply:
I think this is right... I put them in correct alphabetical order.

Reply:
Except these were added in a different PR not this one?